alpha/apps/kaltura/lib/kManifestRenderers.php

```diff
@@ -1029,6 +1029,19 @@ protected function getManifestFlavors()
 		return $flavorsArr;
 	}
 
+	private function shouldUseClosedCaptions(): bool
+	{
+		$dbEntry = entryPeer::retrieveByPK($this->entryId);
+		foreach ($this->contributors as $contributor)
+		{
+			if($contributor instanceof WebVttCaptionsManifestEditor && $dbEntry->getType() == entryType::LIVE_STREAM)
```

Comment:
check that $dbEntry returned before accessing its type elase the manifest can crash just because of the captions.
there is a chance we also check it before this function but i would not take risks here